src/Components/QuestionCard.tsx

```diff
@@ -0,0 +1,219 @@
+import axios from "axios";
+import React, { useEffect, useState } from "react";
+import { FcOk, FcCancel } from "react-icons/fc";
+import { Link, useNavigate } from "react-router-dom";
+import { Option, Question } from "../Data/quizData.types";
+import { useQuiz, useStore, useUser } from "../Store";
+import URL from "./ServerURL";
+import { Toast } from "./Toast";
+
+export const QuestionCard = () => {
+  const [isAnswered, setIsAnswered] = useState(false);
+  const [timeCounter, setTimeCounter] = useState(20);
+  const [isSkipped, setIsSkipped] = useState(false);
```

Comment:
possible to use useReducer here as 3 states being used.

Reply:
Good point. Thanks, I will refactored it to the reducer.